setup.py

```diff
@@ -0,0 +1,21 @@
+from distutils.core import setup
+
+from setuptools import find_packages
+
+setup(
+    name='ebi_eva_common_pyutils',
+    packages=find_packages(),
+    version='0.2.0',
+    license='Apache',
+    description='EBI EVA - Common Python Utilities',
+    url='https://github.com/EBIVariation/eva-common-pyutils',
+    keywords=['EBI', 'EVA', 'PYTHON', 'UTILITIES'],
+    install_requires=['psycopg2', 'requests', 'pymongo', 'lxml'],
```

Comment:
`psycopg2` means I need to have a postgresql server installed to get this module. Could we switch to `psycopg2-binary` instead ?

Reply:
I actually had to install (in ubuntu with apt-get) python3-dev, postgresql-server-dev-10, postgresql-common to be able to get ebi_eva_common_pyutils installed by pip.